pkg/provider/github/github.go

```diff
@@ -238,13 +239,35 @@ func (w *GitHub) HandleHook(ctx context.Context, req *http.Request) (int, error)
 func (w *GitHub) handleEvent(ctx context.Context, client *github.Client, event interface{}, receiver *webhookv1.GitWatcher) (int, error) {
 	execution := initExecution(receiver)
 	switch event.(type) {
+	case *github.CreateEvent:
+		if receiver.Spec.Tag == false {
+			return http.StatusUnprocessableEntity, fmt.Errorf("create events only support tag")
+		}
+		parsed := event.(*github.CreateEvent)
+		if parsed.Ref == nil {
+			return http.StatusUnprocessableEntity, errors.New("create event has invalid tag ref")
```

Comment:
`create event has empty ref` would be more precise?

Reply:
updated